exovetter/lpp.py

```diff
@@ -0,0 +1,330 @@
+#https://github.com/jakevdp/lpproj
+import numpy as np
+from sklearn.neighbors import NearestNeighbors
+from lpproj import LocalityPreservingProjection
```

Comment:
Are sklearn and lpproj now both *required* dependencies or can they be optional? `setup.cfg` needs to be updated.

It is also worrying that `lpproj` hasn't been updated for 4 years. I doubt it will work with latest packages out of the box. We might need to patch it upstream or bundle.

Reply:
If you would like to take it and bundle it with this code, I'd be all for it. But we could kick that can down the road for the moment.
I've never touched a setup.cfg, so I don't trust myself to touch it.
These are only required dependencies if you want to run lpp.   So I'd say they are optional.